packages/i18nify-go/country.go

```diff
@@ -0,0 +1,40 @@
+package i18nify_go
+
+import (
+	metadata "i18nify/packages/i18nify-go/modules/country_metadata"
+	phonenumber "i18nify/packages/i18nify-go/modules/country_phonenumber"
+	subdivisions "i18nify/packages/i18nify-go/modules/country_subdivisions"
+	currency "i18nify/packages/i18nify-go/modules/currency"
```

Comment:
please import as "github.com/i18nify/packages/i18nify-go/modules/country_metadata" just giving path will create import issues outside the package.